src/Aspire.Hosting.Azure.SignalR/AzureSignalRExtensions.cs

```diff
@@ -60,4 +63,62 @@ public static IResourceBuilder<AzureSignalRResource> AddAzureSignalR(this IDistr
         return builder.AddResource(resource)
                       .WithManifestPublishingCallback(resource.WriteToManifest);
     }
+
+    /// <summary>
+    /// Configures an Azure SignalR resource to be emulated. This resource requires an <see cref="AzureSignalRResource"/> to be added to the application model. Please note that the resource will be emulated in <b>Serverless mode</b>.
+    /// </summary>
+    /// <remarks>
+    /// This version of the package defaults to the <inheritdoc cref="SignalREmulatorContainerImageTags.Tag"/> tag of the <inheritdoc cref="SignalREmulatorContainerImageTags.Registry"/>/<inheritdoc cref="SignalREmulatorContainerImageTags.Image"/> container image.
+    /// </remarks>
+    /// <param name="builder">The Azure storage resource builder.</param>
```

Comment:
```suggestion
    /// <param name="builder">The Azure SignalR resource builder.</param>
```

Should check for more copy-pasta.